app/assets/stylesheets/views/_landing_page/hero.scss

```diff
@@ -52,23 +51,18 @@ $large-desktop-height: 400px;
 
 .app-b-hero--mid-page {
   overflow: hidden;
-  min-height: $desktop-height;
 
   @include govuk-media-query($until: desktop) {
     display: flex;
     flex-direction: column-reverse;
   }
 
-  @include govuk-media-query($until: tablet) {
-    min-height: $mobile-height;
-  }
-
   .app-b-hero__textbox-wrapper {
     @include govuk-media-query($from: desktop) {
       box-sizing: border-box;
       align-items: center;
       display: flex;
-      min-height: $desktop-height;
+      height: $desktop-height;
```

Comment:
I'm not 100% sure what the problem being fixed here is but this change introduces an old problem, which is that the text box can get trapped in the wrapper. Before:

![Screenshot 2024-11-20 at 15 11 19](https://github.com/user-attachments/assets/47d368c7-1871-4305-bbdd-9732580f52f5)

With this change:

![Screenshot 2024-11-20 at 15 11 28](https://github.com/user-attachments/assets/b60663dd-0a69-4aff-a61b-a0da1ee16b3d)

Reply:
Thanks for the review and good catch (I'd forgotten about that issue). The problem I'm trying to fix is that hero content seems to be slightly misaligned. I now think the cause is too much padding in the `app-b-hero__textbox-wrapper` container, which causes the hero content to overflow slightly

<img width="1387" alt="Screenshot 2024-11-20 at 16 21 29" src="https://github.com/user-attachments/assets/5c4f7e45-2e1c-4535-a47f-d1a466b9c18f">

I think reducing the padding to 20px should resolve the issue

<img width="1387" alt="Screenshot 2024-11-20 at 16 21 14" src="https://github.com/user-attachments/assets/60b095c8-f37b-47e1-97cc-ab7639b0ba92">

Reply:
I think you could take the padding off entirely for this variant - it doesn't seem to do anything.